Zavier45/CreekRiver
# Campsite

### Purpose
To enable the user to fetch information from the database via endpoints.

### Files Changed
- updated property value in `CreekRiverDbContext.cs`
- updated class references in `CampsiteDTO.cs`, `ReservationDTO.cs`, and `UserProfileDTO.cs`
- added logic for GET, POST, PUT, and DELETE endpoints for campsites and for reservations in `Program.cs`

### To Test
- Run 'dotnet watch run' in terminal and navigate to the swagger pages which opens in the browser.
- Open PGAdmin and open the CreekRiver database.
    - Test each endpoint, inputting the correct information as requested, to verify that each one works and returns the proper information.
    - When testing the 'Post', 'Put', and 'Delete, endpoints, verify the information has updated with the appropriate table in the database.
